lib/transport/api/datagouvfr.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Transport.Api.Base.Datagouvfr do
```

Comment:
Domain: it not's the API, it's the data.gouv.fr client API, so we should change name.